tests/publiccloud/prepare_tools.pm

```diff
@@ -115,16 +115,16 @@ sub run {
     assert_script_run("echo . /root/google-cloud-sdk/path.bash.inc >> ~/.bashrc");
     record_info('GCE', script_output('source ~/.bashrc && gcloud version'));
 
-    my $terraform_version = get_var('TERRAFORM_VERSION', '1.5.7');
-    # Terraform in a container
-    my $terraform_wrapper = <<EOT;
+    my $opentofu_version = get_var('OPENTOFU_VERSION', '1.9.1');
```

Comment:
Why not to put opentofu side by side with terraform for a bit?

Reply:
I agree too, but 1 only shall do the main job, the other one could be `--dryrun` i.e. ?